curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646de7b64467e96b7d35b5cd.md

```diff
@@ -154,9 +154,9 @@ body {
 }
 
 .cat-right-eye {
-  --fcc-editable-region--
-  
-  --fcc-editable-region--
```

Comment:
This is not the requested change, this is breaking the step. Revert this change.